.github/pull_request_template.md

```diff
@@ -1,12 +1,12 @@
-__ISSUENUMBER__ 
```

Comment:
You don't need to edit pull_request_template.md file @coder-raksh2509. Please just write your details in participants.json.